contribs/gmf/src/directives/print.js

```diff
@@ -698,9 +698,13 @@ gmf.PrintController.prototype.handleGetStatusSuccess_ = function(ref, resp) {
   var mfResp = /** @type {MapFishPrintStatusResponse} */ (resp.data);
   var done = mfResp.done;
   if (done) {
-    // The report is ready. Open it by changing the window location.
-    window.location.href = this.ngeoPrint_.getReportUrl(ref);
-    this.resetPrintStates_();
+    if (mfResp.status != 'error') {
+      // The report is ready. Open it by changing the window location.
+      window.location.href = this.ngeoPrint_.getReportUrl(ref);
+      this.resetPrintStates_();
```

Comment:
@fredj is it a good practice to directly use `console.error` or is there an other `closure`-proof way to do it?